onlyswaps-verifier/Dockerfile

```diff
@@ -14,11 +33,12 @@ RUN cargo chef cook --release --recipe-path recipe.json
 
 # Build application
 COPY . .
+COPY --from=sol_onlyswaps /app/onlysubs-solidity/out ./onlysubs-solidity/out
 RUN cargo build --release -p onlyswaps-verifier
 
 # We do not need the Rust toolchain to run the binary!
 FROM debian:bookworm-slim AS runtime
 RUN apt-get update && apt-get install -y libssl3 dnsutils
 WORKDIR /app
 COPY --from=builder /app/target/release/onlyswaps-verifier /usr/local/bin
-ENTRYPOINT ["/usr/local/bin/onlyswaps-verifier"]
+ENTRYPOINT ["/usr/local/bin/onlyswaps-verifier", "--config", "/data/config.json"]
```

Comment:
That part probably warrants a call-out in the README.md about how to run this and how to config it.

Please, add a README.md 🙏🏻

Reply:
README added